dev-aws/kafka-shared-msk/customer-proposition/customer-proposition.tf

```diff
@@ -0,0 +1,29 @@
+resource "kafka_topic" "uswitch.data.v1" {
+  name = "customer_proposition.uswitch.data.v1"
```

Comment:
could you please please use `-` instead? so `customer-proposition.uswitch.data.v1`